backend/workflow_manager/workflow/workflow_helper.py

```diff
@@ -402,13 +403,20 @@ def execute_workflow_async(
             }
             org_schema = connection.tenant.schema_name
             log_events_id = StateStore.get(Common.LOG_EVENTS_ID)
-            async_execution = WorkflowHelper.execute_bin.delay(
-                org_schema,
-                workflow_id,
-                hash_values_of_files=file_hash_in_str,
-                execution_id=execution_id,
-                pipeline_id=pipeline_id,
-                log_events_id=log_events_id,
+            async_execution = WorkflowHelper.execute_bin.apply_async(
+                args=[
+                    org_schema,  # schema_name
+                    workflow_id,  # workflow_id
+                    execution_id,  # execution_id
+                    file_hash_in_str,  # hash_values_of_files
+                ],
+                kwargs={
+                    "scheduled": False,
+                    "execution_mode": None,
+                    "pipeline_id": pipeline_id,
+                    "log_events_id": log_events_id,
+                },
+                queue=queue,
```

Comment:
@muhammad-ali-e how are the celery task's arguments stored in the DB? Do you think this might require a migration for existing tasks?

Reply:
I believe Not needed, Because it directly used by API only.